driver-core/src/main/com/mongodb/internal/operation/CommandOperationHelper.java

```diff
@@ -471,9 +476,8 @@ static <T, R> void executeRetryableWriteAsync(
                     // attach `maxWireVersion`, `retryableCommandFlag` ASAP because they are used to check whether we should retry
                     retryState.attach(AttachmentKeys.maxWireVersion(), maxWireVersion, true)
                             .attach(AttachmentKeys.retryableCommandFlag(), isRetryWritesEnabled(command), true)
-                            .attach(AttachmentKeys.commandDescriptionSupplier(), command::getFirstKey, true)
+                            .attach(AttachmentKeys.commandDescriptionSupplier(), command::getFirstKey, false)
                             .attach(AttachmentKeys.command(), command, false);
```

Comment:
This is similar to https://github.com/mongodb/mongo-java-driver/pull/1108/files?show-viewed-files=true&file-filters%5B%5D=#r1169033282.